### ISSUE TITLE
Build Jar throws NoClassDefFoundError

### ISSUE THREAD

**Issue by MrCraftCod** [MrCraftCod/m3u8Recorder#17]
```
Exception in thread "MainWatcher" java.lang.NoClassDefFoundError: org/apache/http/HttpInetConnection
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(Unknown Source)
        at java.security.SecureClassLoader.defineClass(Unknown Source)
        at java.net.URLClassLoader.defineClass(Unknown Source)
        at java.net.URLClassLoader.access$100(Unknown Source)
        at java.net.URLClassLoader$1.run(Unknown Source)
        at java.net.URLClassLoader$1.run(Unknown Source)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at org.apache.http.impl.conn.PoolingHttpClientConnectionManager$InternalConnectionFactory.<init>(PoolingHttpClientConnectionManager.java:572)
        at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.<init>(PoolingHttpClientConnectionManager.java:174)
        at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.<init>(PoolingHttpClientConnectionManager.java:158)
        at org.apache.http.impl.client.HttpClientBuilder.build(HttpClientBuilder.java:862)
        at fr.mrcraftcod.utils.http.URLHandler.makeClient(URLHandler.java:43)
        at fr.mrcraftcod.utils.http.URLHandler.<clinit>(URLHandler.java:94)
        at fr.mrcraftcod.utils.http.requestssenders.get.GetRequestSender.<init>(GetRequestSender.java:37)
        at fr.mrcraftcod.utils.http.requestssenders.get.GetRequestSender.<init>(GetRequestSender.java:32)
        at fr.mrcraftcod.utils.http.requestssenders.get.JSONGetRequestSender.<init>(JSONGetRequestSender.java:25)
        at fr.mrcraftcod.m3u8recorder.provider.M3U8Provider.getAsJSON(M3U8Provider.java:69)
        at fr.mrcraftcod.m3u8recorder.provider.M3U8Provider.getAsJSON(M3U8Provider.java:54)
        at fr.mrcraftcod.m3u8recorder.provider.LivemeProvider.getProviderInfos(LivemeProvider.java:42)
        at fr.mrcraftcod.m3u8recorder.provider.Providers.getProviderInfosForURL(Providers.java:61)
        at fr.mrcraftcod.m3u8recorder.sourceGetters.SourcesManager.lambda$onNewClipboard$0(SourcesManager.java:30)
        at java.lang.Iterable.forEach(Unknown Source)
        at fr.mrcraftcod.m3u8recorder.sourceGetters.SourcesManager.onNewClipboard(SourcesManager.java:30)
        at fr.mrcraftcod.m3u8recorder.sourceGetters.ClipboardWatcher.lambda$loop$0(ClipboardWatcher.java:113)
        at java.util.ArrayList.forEach(Unknown Source)
        at fr.mrcraftcod.m3u8recorder.sourceGetters.ClipboardWatcher.loop(ClipboardWatcher.java:113)
        at fr.mrcraftcod.utils.threads.ThreadLoop.run(ThreadLoop.java:19)
Caused by: java.lang.ClassNotFoundException: org.apache.http.HttpInetConnection
        at java.net.URLClassLoader.findClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        ... 32 more
```

**Comment by MrCraftCod** [comment]
This happens only with the Jar built from maven

**Comment by MrCraftCod** [comment]
Fixed with MrCraftCod/JavaUtils#06c7abdb0a4d6834ddc8dcd89844448d27157644
